tools/astropytools/fits2csv.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="astropy_fits2csv" name="astropy fits2csv" version="0.1.0+galaxy0" profile="21.05">
+    <requirements>
+        <requirement type="package" version="5.2.2">astropy</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python '$py_script_file' 
+    ]]></command>
+
+    <configfiles>
+        <configfile name="py_script_file">
+from astropy.io import fits
+from astropy.table import Table
+
+with fits.open("$input_fits", memmap=True) as hdu_list:
+
+    # select the HDU you want
+    hdu = hdu_list[$hdu]
+    
+    # read into an astropy Table object
+    table = Table(hdu.data)
+
+    # write to a CSV file
+    table.write('$output_csv', delimiter='$output_delimiter', format='ascii', overwrite=True)
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param type="data" name="input_fits" format="fits" label="FITS file to dump"/>
+        <param type="integer" name="hdu"  value="1" min="1" label="Select input HDU number"/>
+        <param type="select" name="output_delimiter" value="," label="Output delimiter">
+            <option value="," selected="true">"," (coma)</option>
+            <option value=" ">" " (space)</option>
+            <option value="\t">"\t" (tab)</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="output_csv" format="csv"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_fits" value="WFPC2u5780205r_c0fx.fits"/>
+            <param name="hdu" value="1"/>
+            <output name="output_csv" file="fitsinfo.csv"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+Extract CSV table from FITS file HDU Table. Resulting CSV file can be used by many existing Galaxy tools and visualisation plugins.
```

Comment:
Not only CSV this tool can supports more ...

Reply:
I changed the text.